doc/markdown/user/abstraction/Element.md

```diff
@@ -0,0 +1,42 @@
+[:arrow_up: Up](../Abstraction.md)
+
+Element
+=======
+
+To use the maximum available computing power of, for example, a modern x86 processor, the computation has to utilize the SIMD vector registers.
+Many current architectures support issuing a single instruction that can be applied to multiple data elements in parallel.
+
+The original x86 instruction set architecture did not support SIMD instructions but has been enhanced with MMX (64 bit width registers), SSE (128 bit width registers), AVX (256 bit width registers) and AVX-512 (512 bit width registers) extensions.
+In varying degree, they allow to process multiple 32 bit and 64 bit floating point numbers as well as 8, 16, 32 and 64 bit signed and unsigned integers.
+
+*CUDA* capable GPUs do not have vector registers where multiple values of type `float` or `double` can be manipulated by one instruction.
+Nevertheless, newer *CUDA* capable devices implement basic SIMD instructions on pairs of 16 bit values and quads of 8-bit values. 
+They are described in the documentation of the [PTX instruction set architecture](http://docs.nvidia.com/cuda/pdf/ptx_isa_3.2.pdf) chapter 8.7.13 but are only of any use in very special problem domains, for example for deep learning.
```

Comment:
Maybe we should link to [this](http://docs.nvidia.com/cuda/parallel-thread-execution/index.html) online documenation which points always to the newest ISA version